tests/regressions.sh

```diff
@@ -310,17 +313,21 @@ test_timeout_action2() {
 	SBD_TIMEOUT_ACTION=crashdump sbd -d ${D[1]} -w /dev/null -n test-1 watch
 	sleep 2
 	sbd_wipe_disk ${D[1]}
-	sleep 15
+	sleep $((${MSGWAIT_TIMEOUT} + ${WATCHDOG_TIMEOUT} * 2))
 	_in_log "sysrq-trigger ('c')"
 }
 
 sbd_setup
 
+_ok test "${WATCHDOG_TIMEOUT}" -eq "${WATCHDOG_TIMEOUT}"
+_ok test "${MSGWAIT_TIMEOUT}" -eq "${MSGWAIT_TIMEOUT}"
```

Comment:
I'm confused, what does this do?

Reply:
just - little bit dirty admitted - to assure both timeouts are numbers.